draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -507,40 +521,44 @@ the fragment receiver. A single fragmentation rule is defined.
 SCHC F/R MUST concatenate FPort and LoRaWAN payload to retrieve the SCHC
 Packet, as per {{lorawan-schc-payload}}.
 
-* SCHC header size is two bytes (the FPort byte + 1 additional byte).
-* RuleID: 8 bits stored in LoRaWAN FPort.
 * SCHC fragmentation reliability mode: `ACK-on-Error`.
-* DTag: Size is 0 bit, not used.
+* SCHC header size is two bytes (the FPort byte + 1 additional byte).
+* RuleID: 8 bits stored in LoRaWAN FPort. cf {{rule-id-management}}
+* DTag: Size T=0 bit, not used. cf {{DTag}}, §8.2.3. Integrity Checking
```

Comment:
Not sure §8.2.3 is what you intended. Isn't `{{DTag}}` sufficient?

Reply:
Thanks, fixed in dc7f630